### ISSUE TITLE
Error on tap related to mob-timer cask.

### ISSUE THREAD

**Issue by icorbett** [meetup/homebrew-tools#2]
Hey folks, hope everything is treating you well!

`==> Tapping meetup/tools
Cloning into '/usr/local/Homebrew/Library/Taps/meetup/homebrew-tools'...
remote: Enumerating objects: 18, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 60 (delta 6), reused 14 (delta 4), pack-reused 42
Unpacking objects: 100% (60/60), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/meetup/homebrew-tools/mob-timer.rb
mob-timer: undefined method `cask' for Formulary::FormulaNamespace5e3673bd9085ddbaa2281f749d226b3c:Module
Error: Cannot tap meetup/tools: invalid syntax in tap!`

**Comment by chenrui333** [comment]
@icorbett 👋 thanks for getting into my attention, I did actually never make it work. 😅 

Let me try again.

**Comment by chenrui333** [comment]
Fixed it:

```
$ brew cask install mob-timer
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (meetup/tools).
No changes to formulae.

==> Downloading https://github.com/GreatWebGuy/MobTime/releases/download/v1.6.4/MobTime-1.6.4.dmg
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/48971385/499e6a06-43c0
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'mob-timer'.
==> Installing Cask mob-timer
==> Moving App 'MobTime.app' to '/Applications/MobTime.app'.
🍺  mob-timer was successfully installed!
```

But you do need a separate notorization when using it. (right click and open the app should be sufficient)
